CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# the global owners will be requested for
+# review when someone opens a pull request.
+*       @istio/reviewers
+
+# Per component owners. Prefix folders with / so that the owners
+# will be requested for reviews for all subdirectories under that
+# folder in the root
+/.circleci/ @rshriram @costinm # please do not change this
+/release/ @mattdelco
+/prow/ @sebastienvas
+
+## TODO samples, install
+# /pkg/, /tests/, /tools/, /bin/, /docker/ is everyone's responsibility
```

Comment:
How does this work? If there is no entry in the file, who becomes the owner?

Reply:
everyone in Reviewers (i.e. all of us)